yaksh/forms.py

```diff
@@ -150,7 +150,8 @@ def save(self):
             new_profile.key_expiry_time = timezone.now() + \
                                         timezone.timedelta(minutes=20)
         new_profile.save()
-        return u_name, pwd, new_user.email, new_profile.activation_key
+        user_activation_key = Profile.objects.get(user=new_user).activation_key
```

Comment:
Also what happens to <code>user_activation_key</code> if <code>settings.IS_DEVELOPMENT</code> is True? Will it send a None value or will that raise an error?